cloudSetup/variables.tf

```diff
@@ -6,7 +6,7 @@ variable "deputy" {
 
 # Use a short project tag. Long tags will cause the exasol cluster creation to fail (see https://github.com/exasol/cloudformation-aws-exasol/issues/3)
 variable "project" {
-  default = "VS-ADLS"
+  default = "VSADLS"
```

Comment:
Can we use identical abbreviations in [`error_code_config.yaml`](https://github.com/exasol/azure-data-lake-storage-gen2-document-files-virtual-schema/blob/main/error_code_config.yml) and `cloudSetup/variables.tf`?

Reply:
👍